### DISCUSSION TITLE
Let’s discuss why we still don’t have ARM support in Ledger and we all know it’s a big deal

### DISCUSSION THREAD

**Discussion by GokulTheDev** [LedgerHQ/ledger-live-desktop#3940] [General]
Everything moving from x86 to ARM.From apple to Samsung and Microsoft to every major brand but only ledger is not supporting ARM processors compare to other hardware wallets

**Comment by TimmyBoi155** [comment]
Agreed, and it's way more secure

**Comment by GokulTheDev** [comment]
Also ARM processors less vulnerable to spectre and meltdown virus and all X86 viruses created so far
